sysvar: add a note to disable tidb_enable_rate_limit_action (#11444)
pingcap/docs: system-variables.md

### tidb_enable_rate_limit_action

> **Note:**
>

> > This variable is enabled by default. In this situation, the memory usage might not be under the control of [`tidb_mem_quota_query`]. Therefore, it is recommended to modify the variable value to `OFF`.

Author: ```suggestion
> This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](#tidb_mem_quota_query) in some cases. Therefore, it is recommended to modify the variable value to `OFF`.
```

> > This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](#tidb_mem_quota_query) in some cases. Therefore, it is recommended to modify the variable value to `OFF`.

Author: ```suggestion
> This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](#tidb_mem_quota_query) in some cases. Therefore, it is recommended to disable `tidb_enable_rate_limit_action`..
```

> > This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](#tidb_mem_quota_query) in some cases. Therefore, it is recommended to disable `tidb_enable_rate_limit_action`.

Author: ```suggestion
> This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](#tidb_mem_quota_query) in some cases. Therefore, it is recommended to set the value of `tidb_enable_rate_limit_action` to `OFF`.
```
